lddtree.sh

```diff
@@ -24,6 +24,7 @@ usage() {
 	  -b              Force use of specific backend tools (scanelf or binutils)
 	  -R <root>       Use this ROOT filesystem tree
 	  --no-auto-root  Do not automatically prefix input ELFs with ROOT
+	  --no-recursive  Do not recursivly parse dependencies
```

Comment:
What is the feature needed for? Would be nice to have explanation in the commit message why/when this is needed or what value it provides.